src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3827,28 +3816,27 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
     }
 
     void visit_Module(const AST::Module_t &x) {
-        if (!current_scope) {
+        ASR::asr_t *tmp0 = nullptr;
+        if (current_scope) {
+            tmp0 = current_scope->asr_owner;
```

Comment:
Put an assert here ensuring tmp0 is actually a TranslationUnit

Reply:
Added.